packages/shared/lib/services/notification/webhook.service.ts

```diff
@@ -145,6 +149,7 @@ class WebhookService {
                 deleted: 0
             },
             syncType,
+            updatedAfter: dayjs(now).toDate().toISOString(),
```

Comment:
what does dayjs bring that cannot be done with Date?

Reply:
Consistency with how the records.service is handling it is all really.